### PR TITLE
[MGPG-97] use pgpverify plugin to check dependencies signatures

### PR DESCRIPTION
**Pull request by hboutemy**

### CODE REVIEW

**Review by slawekjaranowski — COMMENTED** [review]
We can try.
One of disadvantage can be that we need to update keys mapping when dependency will be updates.

File: pom.xml [hunk 1]
```diff
@@ -194,6 +194,14 @@ under the License.
           <artifactId>maven-invoker-plugin</artifactId>
           <version>3.5.1</version>
         </plugin>
+        <plugin>
+          <groupId>org.simplify4u.plugins</groupId>
+          <artifactId>pgpverify-maven-plugin</artifactId>
+          <version>1.11.0</version>
```
[the remark this answers is not in the document]
Comment (hboutemy) [reply pom.xml]: updated, thank you

**Comment by hboutemy** [comment]
this is part for the experience: if we do not check signature, why sign at all?

**Comment by jira-importer** [comment]
Resolve #163

**Pull request merged**